src/content/docs/en/recipes/reading-time.mdx

```diff
@@ -76,18 +76,18 @@ Create a [remark plugin](https://github.com/remarkjs/remark) which adds a readin
 
     ```astro title="src/pages/posts/[slug].astro" "const { Content, remarkPluginFrontmatter } = await entry.render();" "<p>{remarkPluginFrontmatter.minutesRead}</p>"
```

Comment:
The code snippet uses [Expressive code](https://expressive-code.com/key-features/text-markers/#marking-individual-text-inside-lines) to highlight some parts. So the current text marker no longer matches your edit.
```suggestion
    ```astro title="src/pages/posts/[slug].astro" "const { Content, remarkPluginFrontmatter } = await render(entry);" "<p>{remarkPluginFrontmatter.minutesRead}</p>"
```

Reply:
Expressive Code highlighting has been updated!